test/data/default.js

```diff
@@ -395,7 +395,8 @@ module.exports = models => {
         id: ids.rops.submitted,
         projectId: ids.projects.croydon.activeAA,
         status: 'submitted',
-        year: 2020
+        year: 2020,
+        submittedDate: moment().subtract(1, 'month').toISOString()
```

Comment:
It doesn't look like this will break as we don't check the range in the test, but a hardcoded date <= 31/01/2021 would probably make more sense as it's a 2020 rop :D

Reply:
```suggestion
        submittedDate: '2021-01-19T12:00:00.000Z'
```

Be the change you want to see in the world.